pom.xml

```diff
@@ -70,13 +70,17 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-smallrye-health</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
+      <groupId>com.google.guava</groupId>
```

Comment:
Add Guava dependency for UTF-8 string length calculation method. Looks like some code already depended on Guava but we did not have explicit dependency but transitive one. Safer to have explicit.